src/Proxy/CommonMethodProxy.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Proxy;
+
+use Psr\EventDispatcher\EventDispatcherInterface;
+use Yiisoft\Proxy\ObjectProxy;
+use Yiisoft\Yii\Debug\Collector\CommonServiceCollectorInterface;
+
+class CommonMethodProxy extends CommonServiceProxy
+{
+    private array $methods;
+
+    public function __construct(
+        string $service,
+        object $instance,
+        array $methods,
+        CommonServiceCollectorInterface $collector = null,
+        EventDispatcherInterface $dispatcher = null,
+        int $logLevel = 0
+    ) {
+        $this->methods = $methods;
+        parent::__construct($service, $instance, $collector, $dispatcher, $logLevel);
+    }
+
+    protected function executeMethodProxy(string $method, array $arguments, $result, float $timeStart)
+    {
+        try {
+            if (isset($this->methods[$method])) {
```

Comment:
Let's check it outside of try/catch statement

Reply:
no difference

Reply:
Looks more bulky

Reply:
> Looks more bulky

There is no other way. Or maybe you see it?

Reply:
> There is no other way.

Hmm.. I was research this moment, you are right.
I can suggest you to replace $callable() to `call_user_func`

Reply:
$callable() should be fine.